apps/backoffice-v2/src/pages/Entity/components/EditableDetails/interfaces.ts

```diff
@@ -16,6 +16,7 @@ export interface IEditableDetails {
   valueId: string;
   id: string;
   documents: Array<{
+    propertiesSchema: any;
```

Comment:
This type should be safe to use here.
```suggestion
propertiesSchema: Record<string, unknown>;
```